packages/commons/src/fhir/transformers/utils.ts

```diff
@@ -735,7 +735,7 @@ export function setQuestionnaireItem(
     questionnaire.item = []
   }
 
-  if (label && !questionnaire.item[context._index.questionnaire]) {
+  if (label) {
```

Comment:
How does removing the other condition affect the flow and why was it there originally?

Reply:
When there is just an updated empty custom field, the label of the items wouldn't change but the values would. That generated the orders mismatch of the `fieldId` and `values`. Thus one property's item got swapped with another one which were shown in the original ticket. And why it was in the first place, I'm not sure. But it would make sense i guess when we create a bundle from scratch.